bin/render-activity-stream-html.js

```diff
@@ -304,6 +306,7 @@ function main() { // eslint-disable-line max-statements
     const options = Object.assign({}, baseOptions, {
       direction: getTextDirection(locale),
       locale,
+      title: strings.newtab_page_title,
```

Comment:
Looks like `strings` from the next line can just be reused instead of additionally passing in the title?